src/main/java/com/sonyericsson/rebuild/RebuildSettings.java

```diff
@@ -41,14 +41,16 @@
      * boolean auto rebuild variable.
      */
     private boolean autoRebuild;
+    private boolean autoRebuildDisabled;
```

Comment:
I think this is bad naming and leads at me a bit confused. It suggests that the setting disables the ```autoRebuild``` but the description says something else.